core/modules/replica/ConfigDatabase.h

```diff
@@ -67,8 +67,14 @@ class DatabaseInfo {
     /// @return The names of all tables.
     std::vector<std::string> tables() const;
 
-    std::string directorTable;      // The name of the Qserv "director" table if any.
-    std::string directorTableKey;   // The name of the primary key column in the "director" table.
+    std::string directorTable;          // The name of the Qserv "director" table if any.
+    std::map<std::string,               // The table name (partitioned tables only!).
+        std::string> directorTableKey;  // The name of the table's key representing object identifiers.
```

Comment:
I am a bit confused by language, what does `table's key` mean here? Is it a column name or FK constraint name?

Reply:
Accepted. It was a mistake to use this term. It was introduced in the very first version of the Replication system. Since there are many uses of this term in the code and in the comments to the code then I'd rather fix it in a separate refactoring effort. I have registered it in: https://confluence.lsstcorp.org/display/DM/Fix+incorrect+naming+convention+for+the+object+identifiers